src/uct/api/uct.h

```diff
@@ -184,9 +184,12 @@ enum {
     UCT_IFACE_FLAG_CONNECT_TO_IFACE = UCS_BIT(40), /**< Supports connecting to interface */
     UCT_IFACE_FLAG_CONNECT_TO_EP    = UCS_BIT(41), /**< Supports connecting to specific endpoint */
 
-    /* Thread safety */
-    UCT_IFACE_FLAG_AM_THREAD_SINGLE = UCS_BIT(44), /**< Active messages callback is invoked only from main thread */
-
+    /* active message callback invocation */
+    UCT_IFACE_FLAG_AM_CB_SYNC       = UCS_BIT(44), /**< Active messages callback is invoked only when
```

Comment:
"interface supports active message callback which is invoked only from the context of uct_worker_progress"